### PR TITLE
User api patchmethod

### PR DESCRIPTION
**Pull request by elmenqui**
Still in doubt if the update statement in UserRepository is complete. 
As is, it won't break any other code. 

Next increment will bring the connection between back and front.

### CODE REVIEW

**Review by parpinelopi — COMMENTED** [review]
Looks fine for now. I hope you manage to establish the front-back connection

**Review by parpinelopi — APPROVED** [review]
Looks fine for now. I hope you manage to establish the front-back connection.

File: src/main/java/se/kth/sda6/nysure/user/UserRepository.java [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, String> {
     User findByEmail(String email);
+
+    @Modifying
+    @Transactional
```
Comment (SaraGarabedian) [inline src/main/java/se/kth/sda6/nysure/user/UserRepository.java]: The following lines prevent the app from starting altogether, therefore can't be merged as of now

**Review by SaraGarabedian — CHANGES_REQUESTED** [review]
The changes to UserRepository.java must be revised, so as not to prevent the app from starting

**Review by krismutalapova — CHANGES_REQUESTED** [review]
Same issue as everyone

File: src/main/java/se/kth/sda6/nysure/user/UserRepository.java (hunk 1 above)
Comment (elmenqui) [reply src/main/java/se/kth/sda6/nysure/user/UserRepository.java]: Changes made, now those lines have been deleted as they are not needed.

**Comment by elmenqui** [comment]
Do clone/update the latest branch

**Pull request closed without merging**